README.md

```diff
@@ -35,8 +35,8 @@ Convert the following ActiveRecord sequence to Mongoose:
 ```
 
 ```js
-// Your answer...
-```
+var andy = Instructor.find_by({name: "Andy"})
+var wishlist_items = Wishlist_items.create({description: "Resin Laying Deer Figurine, Gold"})
```

Comment:
Instead we would want to push a new wishlistitem into `andy.wishlistitems`. **Then** we would save the instructor, not the wishlistitem.